library/common/extensions/filters/http/platform_bridge/c_types.cc

```diff
@@ -13,3 +13,10 @@ const envoy_filter_headers_status_t kEnvoyFilterHeadersStatusContinueAndEndStrea
 const envoy_filter_headers_status_t kEnvoyFilterHeadersStatusStopAllIterationAndBuffer =
     static_cast<envoy_filter_headers_status_t>(
         Envoy::Http::FilterHeadersStatus::StopAllIterationAndBuffer);
+
+const envoy_filter_data_status_t kEnvoyFilterDataStatusContinue =
```

Comment:
nit: is the `k` prefix a common c-style convention?

Reply:
It is _a_ convention that has been used in C, C++, Objective-C and Swift. It's the nearest I could come to a style shared by and recognizable across all four languages.

See, for example:
https://google.github.io/styleguide/cppguide.html#Variable_Comments

Note, the Google Style Guide for Objective-C, _no longer_ recommends prefixing global constants with `k` (where it was once quite prevalent), but again I was going for something that would be easily recognizable across all the languages/environments.